capstone/backend/src/main/java/com/google/spanner/LibraryFunctions.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.java.spanner;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.ServiceOptions;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class LibraryFunctions {
+  private static String DATABASE_INSTANCE = "capstone-instance";
+  private static String DATABASE_NAME = "step39-db";
+
+  private static DatabaseClient initClient() {
```

Comment:
I'm thinking about making this a Singleton so that we don't need to create it every time when we need to connect to the database. I did a brief search and think DatabaseClient is thread-safe. https://cloud.google.com/spanner/docs/sessions Can you try it out?